app/styles/components/_global.scss

```diff
@@ -148,3 +148,31 @@ input[type=text] {
   overflow: show;
   padding: 4px;
 }
+
+/* style a regular select element to look like the actions menu */
+select {
+  /* set the dropdown arrow image as the background to simulate the actions menu dropdown */
+  background: url(/images/select-down-arrow.png) 99% / 2% no-repeat $default-background-color;
```

Comment:
When I do that, it wants to pull from 'assets/images/'... Wasn't sure how to fix... I'll look for the solution.